scripts/trim-low-abund.py

```diff
@@ -63,18 +63,9 @@
                          check_valid_file_exists, add_output_compression_type,
                          get_file_writer)
 
-DEFAULT_NORMALIZE_LIMIT = 20
+DEFAULT_TRIM_AT_COVERAGE = 20
 DEFAULT_CUTOFF = 2
```

Comment:
I think it's good to have them local to the script, for comprehension reasons.
Maybe if we make absolutely sure to print them out and indicate where they
are set...